provider/workspace.go

```diff
@@ -83,6 +91,11 @@ func workspaceDataSource() *schema.Resource {
 				Computed:    true,
 				Description: "The access port of the Coder deployment provisioning this workspace.",
 			},
+			"prebuild_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "A computed count, equal to 1 if the workspace was prebuilt.",
```

Comment:
_was prebuilt_, I may be wrong but could we improve terminology here? If my hunch is right both of these are true: "is a prebuild", "was prebuilt"?

Reply:
I've updated the documentation.